draft-ietf-oauth-status-list.md

```diff
@@ -993,9 +993,13 @@ This behaviour may be mitigated by:
 - private relay protocols or other mechanisms hiding the original sender like {{RFC9458}}.
 - using trusted Third Party Hosting, see [](#third-party-hosting).
 
-## Malicious Issuers
+## Issuer Tracking of Reference Tokens
 
-A malicious Issuer could bypass the privacy benefits of the herd privacy by generating a unique Status List for every Referenced Token. By these means, the Issuer could maintain a mapping between Referenced Tokens and Status Lists and thus track the usage of Referenced Tokens by utilizing this mapping for the incoming requests. This malicious behaviour could be detected by Relying Parties that request large amounts of Referenced Tokens by comparing the number of different Status Lists and their sizes.
+A malicious Issuer could bypass the privacy benefits of the herd privacy by 
```

Comment:
I’m not entirely sure if it’s always a malicious issuer. This kind of thing can "accidentally" be introduced by a legitimate issuer due to the various reasons (due to lack of recommendations, knowledge, and whatnot).

It's a bit unfortunate that there isn't much RPs can do about it as URLs such as https://example.com/some_tenant_guid1/statuslists/1 and https://example.com/some_tenant_guid2/statuslists/1 are valid ones.